CSDiscordService/Eval/CSharpEval.cs

```diff
@@ -42,7 +42,8 @@ public class CSharpEval
                 "System.Text.RegularExpressions",
                 "System.Threading",
                 "System.Threading.Tasks",
-                "CSDiscordService.Eval"
+                "CSDiscordService.Eval",
+                "System.Numerics"
```

Comment:
Not in alphabetical order :ree:

Reply:
Oops, can fix that in the next pr :blep: